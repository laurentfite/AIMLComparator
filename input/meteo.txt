météo du jour
météo ce matin
températures cet après-midi
température cette après-midi
temps aujourd hui
météo aujourd hui
temps ce soir
météo dans la soirée
météo à Grenoble ce matin
météo à Paris ce soir
météo à Avignon
temps ce matin à Avignon
la météo ce soir
la température à Paris
ce soir météo
météo soirée à Grenoble
Quelle sera la météo ce soir?
Dis, quelles sont les températures ce matin?
Quel temps fait il dehors?
J'hésite à mettre un manteau, il fait beau dehors?
Quelle est à la météo à Grenoble?
Quelles sont les températures à Grenoble?
Est ce qu'il fait beau à Grenoble?
Quel temps fait il à Grenoble ce soir?
météo grand petit vache
ceci test météo plante
Il est grand temps d'avancer
À quoi ressemble la météo ce matin à Grenoble?
À Grenoble, quelles sont les températures?
Je voudrais la météo pour ce soir
Je veux la météo de Grenoble
Donne moi la météo maintenant
Qu'annonce la météo pour demain?
Quelle température fait il dehors?
Le temps d'aujourd hui annonce quoi?
J'ai raté la météo à la TV, quel temps fait il?
La météo à Grenoble ce soir?
