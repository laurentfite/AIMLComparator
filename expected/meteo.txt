Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau ce soir
Il va faire beau ce soir
Il fera beau aujourd'hui à Grenoble
Ce soir, à Paris, il fera beau!
Il fera beau aujourd'hui à Avignon
Il fera beau aujourd'hui à Grenoble
Il va faire beau ce soir
Il fera beau aujourd'hui à Paris
Il va faire beau ce soir
Ce soir, à Grenoble, il fera beau!
Il va faire beau ce soir
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il fera beau aujourd'hui à Grenoble
Il fera beau aujourd'hui à Grenoble
Il fera beau aujourd'hui à Grenoble
Ce soir, à Grenoble, il fera beau!
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il est grand temps d'avancer
Il fera beau aujourd'hui à Grenoble
Il fera beau aujourd'hui à Grenoble
Il va faire beau ce soir
Il fera beau aujourd'hui à Grenoble
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Il va faire beau aujourd'hui
Ce soir, à Grenoble, il fera beau!
